docs/en/latest/plugins/forward-auth.md

```diff
@@ -117,32 +117,35 @@ We can perform the following three tests.
 1. **request_headers** Send Authorization header from `client` to `authorization` service
 
 ```shell
-$ curl http://127.0.0.1:9080/headers -H 'Authorization: 123'
-{
+curl http://127.0.0.1:9080/headers -H 'Authorization: 123' \
+-d '{
     "headers": {
         "Authorization": "123",
         "Next": "More-headers"
     }
-}
+}'
 ```
 
 2. **upstream_headers** Send `authorization` service response header to the `upstream`
 
 ```shell
-$ curl http://127.0.0.1:9080/headers -H 'Authorization: 321'
-{
+curl http://127.0.0.1:9080/headers -H 'Authorization: 321' \
+-d '{
```

Comment:
ditto

Reply:
good catch! I changed this to two text blocks. PTAL!